### DISCUSSION TITLE
【功能建议】直播信息扩充建议加一个自适应长度功能

### DISCUSSION THREAD

**Discussion by gaoqiaoqijie** [the1812/Bilibili-Evolved#3632] [功能建议 / Ideas]
![image](https://user-images.githubusercontent.com/79361242/186676780-05042eee-476e-4a43-aaf9-c4956e68df0a.png)
直播侧边栏底部只能显示一半up的直播信息，看着很难受。建议做一个显示指定数量up直播信息并且自适应长度的功能，如果能够在设置页面调整显示长度拿就更好了。

**Comment by the1812** [comment]
本来就是自适应的

<div>
<img src="https://user-images.githubusercontent.com/26504152/187211831-24129331-a194-4e57-b8bc-c80ddd180f4a.png" width="300">
<img src="https://user-images.githubusercontent.com/26504152/187211948-343298c7-383b-4647-97f4-40a029c5e103.png" width="300">
</div>

**Reply by gaoqiaoqijie** [reply]
我在用的时候是这样的：如果直播人数小于等于7个，那就是和没开脚本时的样式一样侧边栏能全部显示且不能滚动，但是大于7个的时候左侧的侧边栏长度是固定的，大概会显示7.5条直播信息，这时候就可以滚动查看。
    我说的自适应指的是侧边栏固定显示指定数量的主播信息长度，例如固定显示7个的长度，剩下的滚动查看，避免出现某个主播信息显示不全的情况。也就是我截图中主播“凉风Kaze”只显示了半个头像长度的直播信息

**Reply by the1812** [reply]
我觉得只是你那边的自适应没到底下, 还有一些空间, 正常情况下应该是我图里那样底下只有一小段间距. 要不你导出份设置看看?

**Reply by gaoqiaoqijie** [reply]
[settings.txt](https://github.com/the1812/Bilibili-Evolved/files/9449129/settings.txt)
不支持上传json文件，我重命名成txt了
